src/test/scala/org/ergoplatform/api/routes/ScriptApiRouteSpec.scala

```diff
@@ -42,6 +42,22 @@ class ScriptApiRouteSpec  extends FlatSpec
       |}
       |""".stripMargin
 
+  it should "execute script with context" in {
+    val suffix = "/executeWithContext"
+    val stream = ClassLoader.getSystemClassLoader.getResourceAsStream("execute-script.json")
```

Comment:
There is one problem with this test. It is hard to judge what script is actually executed, and whether the checks actually verify anything.
I suggest to: 
1) compose ErgoLikeContext manually in the test
2) serialize it into Json (relying on ergoLikeContextEncoder from sigma)
3) pass the json to API

Reply:
The sample data for ErgoLikeContext you have sent to me was too large to be hardcoded in the sources. Certainly it is possible to try to reduce it, skipping some non-required fields and reduced sample values. But it is not trivial (I tried to cut tail of large hex strings "0202020202...", but then this request is rejected). And also the less data is present in sample request, the less code coverage we have. This is why I put sample data in resource file. I do not think that it significantly complicates understanding which request is executed (we are not writing all code in the same file, aren't we?)

Reply:
If this file can be parsed in Json means you can do it manually and parse all components one by one.
Then you can create ErgoLikeContext from these components.
Maybe you don't need it here, but you may need in for the equivalence tests.

Reply:
Sorry, I do not understand. There are 3 components in executeWithContext request: script, environment and context. First two are small. Context is large. I can hardcode  script and environment in code and load context from resource file. But frankly speaking I do not understand why it is better. 

It may be useful to construct context from subcomponents  it there are many repeated subcomponents. But in is not the case of ErgoLikeContext, isn't it?

Reply:
This `json` is randomly generated context. So let's leave it like it is.